sdcm/sct_config.py

```diff
@@ -731,6 +737,13 @@ class SCTConfiguration(dict):
                 be provided by the test suite infrastructure.
                 multiple commands can passed as a list"""),
 
+        dict(name="stress_cmd_sst3_verify_delete", env="SCT_STRESS_CMD_SST3_VERIFY_DELETE", type=str_or_list,
```

Comment:
@fruch this "param" and all others that looks like it above, shouldn't be in our SCT config.
(Or at least I don't think they should).
They are specific to tests (and also were changed lately).